src/AcceptanceTests/App_Packages/NSB.AcceptanceTests.6.2.0/DelayedDelivery/When_Deferring_a_message.cs

```diff
@@ -29,7 +29,7 @@ public class When_deferring_a_message : NServiceBusAcceptanceTest
                 .Done(c => c.WasCalled)
                 .Run();
 
-            Assert.GreaterOrEqual(context.ReceivedAt - context.SentAt, delay);
+            Assert.That(context.ReceivedAt - context.SentAt, Is.EqualTo(delay).Within(1).Seconds);
```

Comment:
@Particular/nservicebus-maintainers please see this. The problem is that ASB has its own time on the broker. When clock skew occurs between the client and the broker then this test will fail. Unfortunately it is not even guaranteed that one second is enough. Should we come up with better tests?